frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
+    myFetch = (url, config) =>
```

Comment:
`myFetch` is not really describing the idea. Maybe something like `makeRequest` etc.?